SimPEG/EM/Static/DC/ProblemDC.py

```diff
@@ -422,8 +422,6 @@ def getA(self):
         Grad = self.mesh.nodalGrad
         A = Grad.T * MeSigma * Grad
 
-        Vol = self.mesh.vol
```

Comment:
this isn't used in the nodal prob. just cleaning it out

Reply:
nice.